docs/extend/index.md

```diff
@@ -289,3 +289,20 @@ follows:
 $ docker-runc exec -t f52a3df433b9aceee436eaada0752f5797aab1de47e5485f1690a073b860ff62 sh
 ```
 
+#### Using curl to debug plugin socket issues.
+
+To verify if the plugin API socket that the docker daemon communicates with is responsive, use curl. In this example, we will make API calls from the docker host to volume and network plugins using curl 7.47.0 to ensure that the plugin is listening on the said socket. For a well functioning plugin, then these basic requests should work. Note that plugin sockets are available on the host under `/var/run/docker/plugins/<pluginID>`
```

Comment:
Started to try a different wording, but maybe someone of the docs team has a better suggestion;

```Markdown
You can verify if a plugin's API socket is functioning correctly using `cURL`.
The following example is executed on the docker host that the plugin is installed on,
and uses `cURL` to connect with the plugin socket to performs an API request. The
location of the plugin's socket is based on its ID, and can be found at `/var/run/docker/plugins/<pluginID>`.
```